tests/contracts/blsContracts/Elements.sol

```diff
@@ -0,0 +1,411 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.26;
+
+library Elements {
```

Comment:
Some documentation on what this library provides would be nice. In particular, some information on what an "Element" is, or some link to read up on this.

If this is taken from some third-party source, a line documenting the origin would be good as well.